### DISCUSSION TITLE
Using pnpm as package manager results in wonky import errors. Using yarn works fine.

### DISCUSSION THREAD

**Discussion by mkvlrn-cm42** [discordjs/discord.js#10220] [Q&A]
Decided to give discord.js a try and when installing packages with pnpm I get this error when importing `SlashCommandBuilder`: Module "discord.js" has no exported member SlashCommandBuilder. When using yarn it works just fine.

Importing like this:

```ts
import { SlashCommandBuilder } from 'discord.js';
```

What could be the issue here?

I've had some related issues when using pnpm to install some `@testing-library` packages and in the end I just had to add more devDependencies because pnpm wasn't adding some chained deps for some reason, so I'm guessing this would be the case.

More of a curiosity than anything else, I'll keep using yarn for the time being.

**Comment by Legends-of-NGDB** [comment]
This is your end according to the data you provided. If you have had trouble with another library, then install discord.js the same way you installed `@testing-library` packages.

**Comment by didinele** [comment]
> What could be the issue here?

Considering a lot of us internally use pnpm (and even our monorepo) runs on it, this does sound like it might be on your end 😅

Do you do anything special in your codebase? Like a `.npmrc` file with certain options?
